Getting-Started.md

```diff
@@ -145,4 +145,56 @@ starting/(re)creating/building/attaching containers for a service.
         nofile:
           soft: 65536
           hard: 65536
-        ```
\ No newline at end of file
+        ```
+     
+   
+
+#### CONNECTION REFUSED
```

Comment:
This problem seems to vague and it is partially covered by https://github.com/chaoss/grimoirelab-sirmordred/blob/master/Getting-Started.md#processes-have-conflicts-with-searchguard-

Reply:
I also wanted to emphasise this conflict when not using security guard. I also felt that the title 'connection refused ' too vague.
Basically, I wanted to put the solution of starting with a fresh environment, in case connection error persists in both secure and non-secure versions.
Maybe, we can find better alternative ways to document this solution.
WDYT?

Reply:
I agree with you about finding a better alternative to document this solution. ATM I don't have a clear idea about how to reshape it, since I understand there is no example about a clear error that can be fixed when starting from a fresh environment.

If you have an idea about how to restructure this part, please share it. Otherwise, I would discard this part from the PR and keep the other two.

WDYT?

Reply:
I agree with you about the inability to explain any clear error that can be fixed by a fresh environment. 
Since this is also a doc about getting started and making one comfortable with the system, we could add a small point (before troubleshooting), illustrating how to remove existing dockers and start a fresh one. 
If this sounds good, I can add it there. Otherwise, I'll remove this part and will do the remaining changes. 
WDYT?

Reply:
sounds good! Can we add a section called `How to` and there we can have a Q&A about how to remove existing dockers ? This could go to a different PR.

Reply:
Great! Maybe I can initiate with one QnA, and we'll keep adding.
For this PR, I'll do the necessary changes.